clap_derive/src/utils/ty.rs

```diff
@@ -155,3 +147,19 @@ where
 {
     iter.next().filter(|_| iter.next().is_none())
 }
+
+#[cfg(feature = "unstable-v5")]
+fn get_vec_ty(ty: &Type, vec_ty: Ty, vecvec_ty: Ty) -> Option<Ty> {
+    subty_if_name(ty, "Vec").map(|subty| {
+        if is_generic_ty(subty, "Vec") {
+            vecvec_ty
+        } else {
+            vec_ty
+        }
+    })
+}
+
+#[cfg(not(feature = "unstable-v5"))]
+fn get_vec_ty(ty: &Type, vec_ty: Ty, _vecvec_ty: Ty) -> Option<Ty> {
+    is_generic_ty(ty, "Vec").then(|| vec_ty)
+}
```

Comment:
I feel like this could be simplified with the `cfg!` macro back to the original code which will also make it easier when we remove the feature flag

Reply:
Maybe? I'm not really sure how to do that with if let.

Reply:
Ah, yeah, if-let makes it difficult, nm